docroot/modules/custom/foia_api/src/Plugin/rest/resource/WebformSubmissionResource.php

```diff
@@ -215,6 +215,21 @@ public function post($data) {
         ];
       }
     }
+    if ($webformId == 'wizard_feedback' && empty($errors)) {
+      // Must include a response to either of the three items.
+      $relevance = isset($data['results_relevant_to_search']['How relevant were the results to your search?'])
+        && $data['results_relevant_to_search']['How relevant were the results to your search?'];
+      $expectations = isset($data['results_meet_expectations']['How well do these results meet your expectations?'])
+        && $data['results_meet_expectations']['How well do these results meet your expectations?'];
+      $other_feedback = isset($data['other_feedback']) && $data['other_feedback'];
```

Comment:
Pretty sure these will work:
```suggestion
      $relevance = !empty($data['results_relevant_to_search']['How relevant were the results to your search?']);
      $expectations = !empty($data['results_meet_expectations']['How well do these results meet your expectations?']);
      $other_feedback = !empty($data['other_feedback']);
```

Reply:
Yep!  That works and is much more readable.